example/client/KatranSimpleClient.h

```diff
@@ -92,7 +95,7 @@ class KatranSimpleClient {
 private:
   Vip parseToVip(const std::string &address, uint32_t protocol);
 
-  Real parseToReal(const std::string &address, uint32_t weight);
+  Real parseToReal(const std::string &address, uint32_t weight, uint32_t flags);
```

Comment:
You forgot updating `parseFlags` to `parseVipFlags` and adding `parseRealFlags` below